CHANGELOG.mkd

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 Unreleased
 ----------
 
+- (CODEMGMT-1457) Flip `deploy-spec` setting to `exclude-spec` [#1204](https://github.com/puppetlabs/r10k/pull/1204)
```

Comment:
Since this was never released, I think we can just document the new `exclude-spec` setting.

Reply:
You gonna update this?

Reply:
It is unclear what you are asking for in this comment...what are you asking for?

Reply:
Change the release note to document the new `exclude-spec` feature without mentioning `deploy-spec`. There is no released version with `deploy-spec`, so mentioning it could be confusing to users.

Reply:
I don't think this would cause user confusion; we already have another release note that mentions `deploy_spec` within this same stanza of unreleased changes, two lines down from this change.

Reply:
oh, I missed that since it wasn't wrapped in backticks. I think we should remove that note too then. Those seem like development steps that don't need to be in the changelog. People can refer to commit messages if they want that level of detail.

I think the notes for CODEMGMT-1421, CODEMGMT-1422, and this one should all be consolidated into a single note about the new `exclude-spec` setting.